### PR TITLE
chatterino2: 2.1.7 -> 2.2.2

### PR DESCRIPTION
**Pull request by bbigras**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

/nix/store/kb9fqca9sy62537pig7hjbdnffdsda04-chatterino2-2.1.7    448.8M
/nix/store/vpxp3j7gb4rvld3gj4bjjjzqlp1gry95-chatterino2-2.2.2    449.1M

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/chatterino2/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 mkDerivation rec {
   pname = "chatterino2";
-  version = "2.1.7";
+  version = "2.2.2";
   src = fetchFromGitHub {
     owner = "fourtf";
```
Comment (cole-h) [inline pkgs/applications/networking/instant-messengers/chatterino2/default.nix:7]: The repo is now hosted at https://github.com/Chatterino/chatterino2, so I think this should be updated to reflect that (as well as the `meta.homepage` below).

```suggestion
    owner = "Chatterino";
```

**Comment by drewrisinger** [comment]
If you're updating, this package could use a little bit of formatting (e.g. ``nix-shell -p nixpkgs-fmt --run "nixpkgs-fmt ./pkgs/.../chatterino2/default.nix"``. Could also add ``meta.changelog = "https://github.com/Chatterino/chatterino2/blob/master/CHANGELOG.md";``

**Review by cole-h — APPROVED** [review]
Diff LGTM (provided minor nit below is addressed). Also, worketh for me:

Result of `nixpkgs-review pr 97914` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>chatterino2</li>
  </ul>
</details>

**Comment by bbigras** [comment]
Done. Thanks.

**Comment by bbigras** [comment]
@marsam help merging this please

**Comment by bbigras** [comment]
Thank you very much

**Pull request merged**